src/microbe_stage/editor/EditorComponentBottomLeftButtons.cs

```diff
@@ -241,8 +252,6 @@ private void OnNameTextEntered(string newText)
                 speciesNameEdit.GetNode<AnimationPlayer>("AnimationPlayer").Play("invalidSpeciesNameFlash");
             }
         }
-
-        EmitSignal(nameof(OnNameSet), newText);
```

Comment:
I feel like this is a bit of a problem because for example the multicell editor has UseSpeciesNameValidation set to false and it wants to do its own validation so not having any signals triggered from this method means that it can't be made to react to this.

Reply:
The validation method itself does not have it, but the ones that call it (other than the one that does not change the text) have it.

Reply:
A problem I do see, is it won't release focus with UseSpeciesNameValidation false. I guess I could fix that bug too.